core/src/either.rs

```diff
@@ -292,11 +304,15 @@ where
     }
 }
 
-/// Implements `Future` and dispatches all method calls to either `First` or `Second`.
+/// Dispatches all method calls to either a `First` or `Second` `Future`, each
+/// of which contains `EitherOutput` that contains the `Item` of `First` or
```

Comment:
`EitherOutput` is not in each of `First` or `Second` but the `Item` yielded by `EitherFuture`'s `Future` implementation.